### DISCUSSION TITLE
Compile open3D and generate respective .so files to be used in android application.

### DISCUSSION THREAD

**Discussion by bingkangzhang** [isl-org/Open3D#3798] [Q&A]
I want to build open3D from source to be used in android app, but failed. I need a sample android app with open3d integrated into it. Any link or resources will be a really great help.Thanks a lot.

**Comment by supersumepro7** [comment]
IStill havent gotten that deeb into the whole open3D project but ill get
back to you as soon as i find a good link.

**Comment by germanros1987** [comment]
At the moment we don't support Android development.
